tools/test.py

```diff
@@ -47,7 +47,10 @@
 
 from os.path import join, dirname, abspath, basename, isdir, exists
 from datetime import datetime
-from Queue import Queue, Empty
+try:
+    from Queue import Queue, Empty # Python 2
+except ModuleNotFoundError:
+    from queue import Queue, Empty # Python 3
```

Comment:
```python
try:
    from queue import Queue, Empty  # Python 3
except ImportError:
    from Queue import Queue, Empty  # Python 2
```

Reply:
https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection